javascript/javascript1/week3/cactus.js

```diff
@@ -0,0 +1,43 @@
+//CactusIO-interactive (Smart phone usage app) 
+const activities = [];
+
+
+function addActivity(date, activity, duration) {
+
+    activities.push({ date: date, activity: activity, duration: duration })
+
+}
+
+addActivity("23/7-18", "Youtube", 30);
+addActivity("23/7-18", "Facebook", 40);
+addActivity("23/7-18", "Netflix", 120);
+addActivity("23/7-18", "Slack", 200);
+addActivity("23/7-18", "dr.dk", 60);
+
+
+
+
+
+
+console.log(activities);
+
+
+function showStatus(value) {
+    let totalDuration = 0;
+    if (value.length < 1) {
+        console.log("Add some activities before calling showStatus")
+    }
+
+    for (i = 0; i < value.length; i++) {
+        totalDuration = totalDuration + value[i].duration;
+
+    }
+    if (totalDuration > 400) {
+        console.log("You have reached your limit, no more smartphoning for you!");
```

Comment:
Do you want to show the warning when `totalDuration` is 400 or more, or do you want to show it when `totalDuration` is 401 or more?
```suggestion
    if (totalDuration >= 400) {
        console.log("You have reached your limit, no more smartphoning for you!");
```

Reply:
I read the homework again and see that it asks to check "usage is above the limit he set", so your original code is correct. :)

Reply:
Thankyou for the feedback

Reply:
No worries, Slavia. Good coding.